libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -128,34 +86,9 @@ public void submitVC(VerifiableCredential vc, String govukSigninJourneyId, Strin
 
         LOGGER.info(LogHelper.buildLogMessage("Sending VC to CIMIT."));
         try {
-            if (configService.enabled(CIMIT_API_GATEWAY_ENABLED)) {
-                var payload =
-                        OBJECT_MAPPER.writeValueAsString(new PostCiApiRequest(vc.getVcString()));
-
-                sendPostHttpRequest(POST_CI_ENDPOINT, govukSigninJourneyId, ipAddress, payload);
-
-            } else {
-                var payload =
-                        OBJECT_MAPPER.writeValueAsString(
-                                new PutCiRequest(
-                                        govukSigninJourneyId, ipAddress, vc.getVcString()));
-
-                var invokeRequest =
-                        InvokeRequest.builder()
-                                .functionName(
-                                        configService.getEnvironmentVariable(
-                                                CI_STORAGE_PUT_LAMBDA_ARN))
-                                .payload(SdkBytes.fromUtf8String(payload))
-                                .qualifier(LIVE_ALIAS)
-                                .build();
-
-                var response = lambdaClient.invoke(invokeRequest);
-
-                if (lambdaExecutionFailed(response)) {
-                    logLambdaExecutionError(response, CI_STORAGE_PUT_LAMBDA_ARN);
-                    throw new CiPutException(FAILED_LAMBDA_MESSAGE);
-                }
-            }
+            var payload = OBJECT_MAPPER.writeValueAsString(new PostCiApiRequest(vc.getVcString()));
+
+            sendPostHttpRequest(POST_CI_ENDPOINT, govukSigninJourneyId, ipAddress, payload);
         } catch (JsonProcessingException e) {
             throw new CiPutException("Failed to serialize payload for post CI request.");
```

Comment:
One thing that Patrick spotted (https://gds.slack.com/archives/C05TJ52FZ8B/p1724411977038429) is that this error can also be thrown when deserializing the response and the error message is quite misleading.

The other thing from both these blocks is that we're losing the underlying exception, so we should make sure any relevant detail has been logged.